models/staging/edfi_3/base/base_ef3__peoples.sql

```diff
@@ -0,0 +1,20 @@
+with peoples as (
```

Comment:
`people` is already plural

Reply:
It should match what the API is.  and the API is /ed-fi/people

Reply:
@maxalmazov @holdorph that makes sense. I got caught 😅